### PR TITLE
Make subscription id optional

### PR DESCRIPTION
**Pull request by ndowmon**
This change allows users to set `ingestResourceManager` to `false`.

### CODE REVIEW

File: src/getStepStartStates.ts [hunk 1]
```diff
@@ -115,4 +113,18 @@ export default function getStepStartStates(
     [STEP_RM_CONTAINER_REGISTRIES]: resourceManager,
     [STEP_RM_CONTAINER_REGISTRY_WEBHOOKS]: resourceManager,
   };
+
+  executionContext.logger.info(
```
Comment (austinkelleher) [inline src/getStepStartStates.ts:117]: Did you intend for this to be exposed to the customer? You have to use `logger.publishEvent` to expose to customer integration log.

**Review by austinkelleher — APPROVED** [review]
Besides minor question about logging, this looks good to me. Thanks for fixing!

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,8 +8,11 @@ and this project adheres to
 
 ## [Unreleased]
 
+## 5.2.0 - 2020-09-20
+
 ### Added
 
+- Added `ingestResourceManager` flag
```
Comment (austinkelleher) [inline CHANGELOG.md:15]: I think it would beneficial to explain what this is used for in the CHANGELOG

File: src/getStepStartStates.ts (hunk 1 above)
Comment (ndowmon) [reply src/getStepStartStates.ts:117]: Yeah, that was a bit of an oversight. I agree with what you said in https://github.com/JupiterOne/graph-fastly/pull/3#pullrequestreview-492570290 so I can remove this log statement in a future iteration.

File: CHANGELOG.md (hunk 2 above)
Comment (ndowmon) [reply CHANGELOG.md:15]: In the spirit of getting this change out, I'll explain this flag in the upcoming https://github.com/JupiterOne/graph-azure/pull/161.

**Review by aiwilliams — COMMENTED** [review]
I'd like to get https://github.com/JupiterOne/graph-microsoft-365/tree/initial-release moving forward so that folks can install a Microsoft 365 integration. I'll add an Epic for this so we can prioritize alongside other work.

**Pull request merged**